gcp/modules/gcp-stackdriver-monitoring/alert_k8s_snapshots_couchdb.tf

```diff
@@ -2,39 +2,47 @@ resource "google_monitoring_alert_policy" "k8s_snapshots_couchdb" {
   display_name = "Snapshots are being created for persistent volumes of CouchDB stateful set"
   combiner     = "OR"
 
-  conditions = [{
-    condition_absent {
-      filter   = "metric.type=\"logging.googleapis.com/user/k8s_snapshots.couchdb.snapshot_created\" resource.type=\"k8s_container\""
-      duration = "300s"
-
-      aggregations {
-        alignment_period   = "600s"
-        per_series_aligner = "ALIGN_SUM"
-        group_by_fields    = []
-      }
-    }
-
-    display_name = "Snapshot creation events are missing in K8s-snapshot logs for CouchDB"
-  },
+  conditions = [
     {
       condition_absent {
         filter   = "metric.type=\"logging.googleapis.com/user/compute.disks.createSnapshot\" resource.type=\"gce_disk\""
         duration = "300s"
 
         aggregations {
-          alignment_period   = "600s"
+          alignment_period   = "300s"
           per_series_aligner = "ALIGN_SUM"
           group_by_fields    = []
         }
       }
 
       display_name = "Snapshot creation events are missing for CouchDB GCE disks"
     },
+    {
+      condition_threshold {
+        filter = "metric.type=\"logging.googleapis.com/user/compute.disks.createSnapshot\" resource.type=\"gce_disk\""
+
+        comparison      = "COMPARISON_LT"
+        threshold_value = 1.0
+        duration        = "300s"
```

Comment:
The metric never goes below threshold  for both ephemeral and static environments in case `k8s-snapshots` operates normally. I didn't notice any flapping of the alert policy that deployed in my dev environment.

Reply:
I reacted below, but marking this as resolved since it's been increased to `10m`.